test/steps/data.go

```diff
@@ -93,26 +93,25 @@ func (data *Data) AfterScenario(s interface{}, err error) {
 		return nil
 	})
 
-	handleScenarioResult(data, s, err)
-	logScenarioDuration(data, s)
+	handleScenarioResult(data, pickle, err)
+	logScenarioDuration(data, pickle)
 	deleteTemporaryExamplesFolder(data)
 }
 
-func logScenarioDuration(data *Data, s interface{}) {
+func logScenarioDuration(data *Data, pickle *messages.Pickle) {
```

Comment:
seems that pickle parameter can be removed here, it doesn't seem to be used.

Reply:
deleted